src/script-loader.ts

```diff
@@ -22,12 +26,18 @@ export default class ScriptLoader {
         private _requestSender: RequestSender
     ) {}
 
-    loadScript(src: string, options?: LoadScriptOptions): Promise<void> {
+    loadScript(src: string, options?: LoadScriptOptions, scriptAttributes?: ScriptAttributes): Promise<void> {
```

Comment:
Just add the additional attributes as a part of `LoadScriptOptions` instead of introducing a third parameter.

Reply:
Done @davidchin